packages/expo-haptics/src/ExpoHaptics.web.ts

```diff
@@ -0,0 +1,47 @@
+import { NotificationFeedbackType, ImpactFeedbackStyle } from './Haptics.types';
+
+/**
+ * The vibrate pattern holds an array of values describes alternating periods in which the device is
+ * vibrating and not vibrating. Each value in the array is converted to an integer, then interpreted
+ * alternately as the number of milliseconds the device should vibrate and the number of
```

Comment:
```suggestion
 * alternately as the duration of milliseconds the device should and should not vibrate.
```

Let's use duration here since millisecond represents a time interval here. Also, let's be concise and remove the last part since the duration is specified here already.